src/rrd_graph.c

```diff
@@ -6147,3 +6155,75 @@ void time_clean(
     }
     result[jj] = '\0'; /* We must force the end of the string */
 }
+
+char *substring(const char *string, int position, int length)
+{
+   char *pointer;
+
+   pointer = malloc(length+1);
+
+   if (pointer == NULL)
+   {
+      printf("Unable to allocate memory.\n");
+      exit(1);
+   }
+
+   strncpy(pointer, (string+position), length);
+   *(pointer+length+1) = '\0';
+
+   return pointer;
+}
+
+image_title_t graph_title_split(
+    const char *title)
+{
+    image_title_t retval;
+    int start = 0;
+    int length = 0;
+    int pos = 0;
+    int count = 0;
+    int delim = 1;
+
+    retval.lines = malloc((MAX_IMAGE_TITLE_LINES + 1 ) * sizeof(char *));
+    length = strlen(title);
+
+    char *delims[6] = { "\n", "\r", "\\n", "\\r", "<br>", "<br/>" };
```

Comment:
there is already the option to use simple markup in rrdtool strings ... so in that way <br> is a good idea ... bringing \r into the mix opens a can of worms I would rather not want to touch ... please stick to \n since this is what most languages will use for breaking a line ...

Reply:
OK, I'll remove \r and try to find this rrdgraph.pod (whatever/wherever that may be 👍 )

Reply:
Got it, in /docs

Reply:
OK this has now been done.